src/spyglass/position/v1/position_dlc_pose_estimation.py

```diff
@@ -409,6 +409,16 @@ def fetch_dataframe(self, *attrs, **kwargs) -> pd.DataFrame:
             axis=1,
         )
 
+    def fetch_video_path(self):
+        """Return the video path for pose estimate
```

Comment:
Normally, we pass key as an arg for this kind of thing. It's not clear to me why you break the pattern here. If no arg, it could be a property, yeah? If kept, I'd add example use to the docstring

```suggestion
        """Return the video path for pose estimate
        
        >>> (DLCPoseEstimation & key).fetch_video_path()
```